connector/rest/src/main/java/it/pagopa/selfcare/onboarding/connector/rest/client/PartyProcessRestClient.java

```diff
@@ -49,16 +48,16 @@ OnBoardingInfo getOnBoardingInfo(@RequestParam(value = "institutionExternalId",
 
     @GetMapping(value = "${rest-client.party-process.getInstitutionByExternalId.path}", produces = APPLICATION_JSON_VALUE)
     @ResponseBody
-    Institution getInstitutionByExternalId(@PathVariable("externalId") String externalId);
+    it.pagopa.selfcare.onboarding.connector.rest.model.Institution getInstitutionByExternalId(@PathVariable("externalId") String externalId);
```

Comment:
Puoi chiamarla direttamente InstitutionResponse, in questo modo non hai l'omonimità sul nome delle classi e non devi mettere tutto il path del modello